src/components/pages/Theme/Theme.jsx

```diff
@@ -21,7 +24,7 @@ import {
 } from 'semantic-ui-react';
 
 // styles
-import './Theme.css';
+import styles from './Theme.module.css';
```

Comment:
I thought we agreed on not using css modules?

Reply:
Our styles were conflicting with the styles in the Redux DataGrid, so as a precaution, we're using modules, as not to interfere with Semantic, Redux Datagrid or any other packages.